assets/stylesheets/layouts/_default.scss

```diff
@@ -3,4 +3,12 @@
   .app-header {
     @apply w-full mx-auto px-4 sm:px-6 lg:px-8;
   }
+
+  .app-header {
+    @apply relative bg-gray-300 mb-4 p-4 text-right;
+
+    &__link {
```

Comment:
This should be part of the `app-header` component. In layout files, only keep styles that affect the layout rendering.

Reply:
@olivierobert ✅  moved on [a945541](https://github.com/carryall/go-google-scraper-challenge/pull/32/commits/a945541b662b58c1f0c51ca0e2826edf7e167beb)